api/src/singleFunction/removeContainer.ts

```diff
@@ -24,7 +32,10 @@ export async function removeContainer({
     await container.remove({ force: !!kill });
 
     const mountedVolumes = containerInfoFull.Mounts.filter(
-      (mount) => mount.Type === 'volume' && mount.Name
+      (
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        mount: any /** TODO any to workaround issue in @types/dockerode https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71974 */
```

Comment:
depending on the review of the DefinitelyTyped PR we could address this TODO in this PR